### ISSUE TITLE
Bug: Pixi + Spine. There is an incorrect behavior of some bones

### ISSUE THREAD

**Issue by brOlk** [pixijs/spine#499]
Hi! We have run into an issue.
Some spine clips of version 3.8 do not work correctly when switching to the new version of PIXI 7. But everything works correctly when using a similar spine clip of version 4.1 and PIXI 7+ or using a spine clip of version 3.8 and any PIXI 6 sub-version.
I attach a demo project with the implementation of the three described situations.

This is a demo project with the described situations.
[demo-project.zip](https://github.com/pixijs/spine/files/11379086/demo-project.zip)

Environment
- pixi.js version: 7.1.0
- Browser & Version: Chrome 112
- OS & Version: Windows 10

**Comment by ivanpopelyshev** [comment]
I'll look at it :)

**Comment by ivanpopelyshev** [comment]
no, still no time to look at that :( Can anyone else do that please? Ask in our discord https://discord.gg/2N5Pd8X4

**Comment by miltoncandelero** [comment]
> May I ask, is there any progress on the issue?

Try loading the 3.8 file with the 4.1 runtime. It will complain but should work anyway.

That's how we handled it before moving to `spine-pixi`

**Comment by anosu** [comment]
Is there any progress on the issue? I encountered the same problem.

pixi.js version: 6.5.10
pixi-spine version: 3.1.2

When loading with runtime 3.8:
![Image](https://github.com/user-attachments/assets/d56e10df-0819-4dcb-9aaf-67c30b2dede0)

Load it with runtime 4.1, it seems that some things are not displayed, for example, the facial expressions are missing here.
![Image](https://github.com/user-attachments/assets/ff66c38b-7239-49aa-8e37-73e665d15d45)

The similar condition occured when I load many other 3.8 (3.8.95/3.8.99) skel files.
I've tried pixi.js 7 & latest pixi-spine with runtime 3.8 but got same result.

In addition, there is no problem when loading with [skeleton-viewer](https://en.esotericsoftware.com/spine-skeleton-viewer) and official spine-ts 3.8

Sample files: [Theme.zip](https://github.com/user-attachments/files/19553552/Theme.zip)
the json file is converted from skel.

**Comment by ivanpopelyshev** [comment]
Interesting, I thought that one was fixed, i just forgot to close that issue

**Comment by ivanpopelyshev** [comment]
Unfortunately i cant look inside model through editor right now, can you clarify, is it clip-related?

**Comment by anosu** [comment]
I'm sorry I dont know.
I dont use spine pro myself, I'm not very familiar with spine itself, I just use runtime to play out other people's files.

**Comment by anosu** [comment]
If you means the animation, it seemes unrelated. After loading the file, it is like this regardless of whether I play the animation or not
